webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js

```diff
@@ -115,6 +119,8 @@ PageLayout.propTypes = {
 };
 
 PageLayout.defaultProps = {
+  className: '',
```

Comment:
Should this default to `undefined` so we don't get an empty prop when it isn't used?

Reply:
I think that passing `undefined` will still result in `<div class />` but didn't check,
We can do something like:
```js
const cxProps = className ? {} : { className };
// later in div
<div {...cxProps} />
```
but I wonder if it really worth the effort?

Reply:
Actually I think react removes `undefined` and `null` attributes:
https://github.com/facebook/react/blob/8b88ac2592c5f555f315f9440cbb665dd1e7457a/packages/react-dom/src/shared/DOMProperty.js#L143-L152

Reply:
Awesome, will change to null then!